.github/workflows/ci.yml

```diff
@@ -44,16 +44,8 @@ jobs:
         run: |
           python -c "import nlmod; nlmod.util.download_mfbinaries()"
 
-      - name: Run notebooks
-        if: ${{ github.event_name == 'push' }}
-        env:
-            NHI_GWO_USERNAME: ${{ secrets.NHI_GWO_USERNAME}}
-            NHI_GWO_PASSWORD: ${{ secrets.NHI_GWO_PASSWORD}}
-        run: |
-          py.test ./tests -m "not notebooks"
-
       - name: Run tests only
-        if: ${{ github.event_name == 'pull_request' }}
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'push'}}
```

Comment:
The if statement can be removed. The workflow is triggered on pr and push, so which other events do you exclude with this line?

Reply:
good point!